examples-extra/docs_example/ontology.json

```diff
@@ -332,7 +332,7 @@
             "description": "TimeSeries of the status of the employee",
             "dataType": {
               "type": "timeseries",
-              "itemType": "string"
```

Comment:
This JSON was incorrect before, so when we converted to timeseries prop we hit the fallback and made this `numericTimeseries` in our generated code. This is now fixed, so the generated code also reflects that